src/test/scala/au/com/agiledigital/toolform/app/GenerateTaskTest.scala

```diff
@@ -12,7 +12,7 @@ class GenerateTaskTest extends FlatSpec with Matchers {
   val testFile = pathToFile("/test_project/environment.conf")
 
   def pathToFile(pathToFile: String): File = {
```

Comment:
Rename file - it is a test for the Docker V3 generation.

I'd consider having folders of input files and expected output files that are picked up by the test automatically when a new case is added.

Reply:
The tests now scans the `src/test/resources/testprojects` directory and for each directory it looks for an `environment.conf` file as the input and an `expected.yaml` as the output.

It ignores comments because they will change according to version/environment/time and date.

I put a README in the directory to explain this for future developers.